.gitignore

```diff
@@ -2,8 +2,6 @@
 
 .eslintcache
 
-.env
```

Comment:
No. What does the .gitignore do?

Reply:
It chooses which files to ignore committing the project onto the repository